src/write-archive/index.test.ts

```diff
@@ -11,13 +11,13 @@ const snapshotJson = {
     {
       type: NodeType.Element,
       attributes: {
-        src: '/home/',
+        src: 'http://localhost:3000/home/',
       },
     },
     {
       type: NodeType.Element,
       attributes: {
-        src: '/img?src=some-path',
+        src: 'http://localhost:3000/img?src=some-path',
```

Comment:
Do we save the URLs like this in the archives or as relative paths? If we save as absolutes, that would be a non-starter.

Reply:
The asset archive is built by watching network requests, so they are always the full URL there. This change makes it easier to match the assets from the archive with the attributes in the DOM since both sides will be full URLs now.

Reply:
Let me rephrase my question.

What happens when you render in the Capture Cloud? Do the archives try to pull from a relative path or http://localhost:3000? The localhost lookup will not work in the capture cloud so those need to be replaced by relative paths

Reply:
Ah yeah, they are relative in the archive storybook so that they load from the capture cloud. Otherwise all the images in [the build](https://www.chromatic.com/build?appId=653fef099b8957739e7534a4&number=21) would be broken!